_docs/platforms/kubernetes/install/1.10.md

```diff
@@ -9,44 +9,62 @@ redirect_from: /docs/install/kubernetes
 
 # Kubernetes 1.10+
 
+{% include operator/install.md %}
+
+## Custom Installations
+
+There is a variety of flavours, versions and particularities in the container orchestrator scope.
+Because of that, StorageOS installation procedures aims to be flexible so it can fit different needs
+regarding the environment, preferences or requirements. The StorageOS cluster operator makes the
+installation easier by delegating the responsibility of the installation to its logic. If you want
+to see and alter the installation of StorageOS, the following examples give you that capacity. Feel
+free to extend and modify the publicly available examples.
+
+### Manifests based
+
+The following github repository hosts installation examples.
+
 ```bash
-# Install StorageOS as a daemonset with RBAC support
 git clone https://github.com/storageos/deploy.git storageos
-cd storageos/k8s/deploy-storageos/standard
-./deploy-storageos.sh
+cd storageos/k8s/deploy-storageos
 ```
 
-or using the Helm chart:
+You can see examples available such as `standard`, `CSI (Container Storage Interface)` or
+`etcd-as-svc`. All of them has a `deploy-storageos.sh` that serves as a wrapper to trigger the
+manifest creation. Follow the according `README.md` of each one for more details.
+
+### Helm installation
+
+StorageOS can be installed with Helm. Helm adds versatility on the installation method. By editing
+the `values.yaml` of the Chart, different combinations of parameters can be passed to deploy
+StorageOS.
+
 ```bash
 git clone https://github.com/storageos/helm-chart.git storageos
 cd storageos
-# Set cluster.join to hostnames or ip addresses of at least one node
-helm install . --name my-release --set cluster.join=node01,node02,node03
-
-# Follow the instructions printed by helm install to update the link between Kubernetes and StorageOS. They look like:
-$ ClusterIP=$(kubectl get svc/storageos --namespace storageos -o custom-columns=IP:spec.clusterIP --no-headers=true)
-$ ApiAddress=$(echo -n "tcp://$ClusterIP:5705" | base64)
-$ kubectl patch secret/storageos-api --namespace storageos --patch "{\"data\":{\"apiAddress\": \"$ApiAddress\"}}"
 ```
 
-## Installation with CSI (Container Storage Interface)
+The following examples show how can you deploy StorageOS in a standard fashion or using CSI.
+
+1. Standard
+    ```bash
+    # Set cluster.join to hostnames or ip addresses of at least one node
+    helm install . --name my-release --set cluster.join=node01,node02,node03
```

Comment:
Shouldn't we be using the new charts repo instead of cloning the git repo and installing this way?

Reply:
Yes, but I wanted to make sure that the new repo will work with previous versions. I know we have to use the right version of the Chart, but I haven't tested yet. I would like to do so before finishing this PR. So I can merge with the new Charts repo.

Reply:
I tested and I'm applying a change to use the new repo.